server/dal/stock-info-old.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const dotenv = require('dotenv');
+dotenv.config();
+
+var Schema = mongoose.Schema
+
+var stockInfoSchema = new Schema({
+    company_name: String,
+    ticker: String,
+    current_price: Number,
+    previous_close: Number,
+    open: Number,
+    last_price: Number,
+    change: Number,
+    change_percent: String,
+    currency: String,
+    volume: String,
+    pe_ratio: Number,
+    shares_owned: Number,
+    avg_vol: String,
+    market_cap: String,
```

Comment:
Have we decided on using strings for large number values? I think we talked a little about using the number type in the backend and then have the frontend truncate the number for viewing format purposes.